proposals/0282-record-dot-syntax.rst

```diff
@@ -468,6 +500,21 @@ forms of update both of which fail in different corner cases. Instead,
 we recommend use of ``C{foo}`` as a pattern (with ``-XNamedFieldPuns``)
 to extract fields if necessary.
 
+7.7 Why two extensions and not just one?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Having two extensions provides fine grained control. For example, one
+can enable the use of ``.`` for field selection without affecting
+record updates. Conversely, one can enable overloaded record updates
+without giving ``.`` new meaning.
```

Comment:
This looks like a pretty weak motivation to have two extensions. After all, extensions have a cost, so we must get a real benefit for that cost.

That being said, my understanding of the thread is that there are, indeed, stronger motivations. At least the fact that the record update syntax is not backward compatible should be brought up. I'm not entirely sure I managed to understand what motivated everyone involved. But it's alright: that's what this section is about, it should document the motivations which have accumulated through the threads (and objections as well). That would be really helpful!